Spanner/tests/Unit/DatabaseTest.php

```diff
@@ -287,19 +307,22 @@ public function testDrop()
      */
     public function testDropDeleteSession()
     {
-        $this->connection->createSession(Argument::any())
+        $this->connection->createSession(Argument::withEntry('database', self::$databaseName))
             ->shouldBeCalled()
             ->willReturn([
-                'name' => SpannerClient::sessionName(self::PROJECT, self::INSTANCE, self::DATABASE, self::SESSION)
+                'name' => self::$sessionName
```

Comment:
Why not `$this->session->name()`?

Reply:
Yes, looks like these calls are equivalent. Replacing.